04-concept-api/00-overview.md

```diff
@@ -19,7 +19,9 @@ Anything in Grakn, whether a concept type or a data instance, is a [Concept](../
 
 <div class="note">
 [Important]
-The methods called on Concepts are bound to the [transaction](../03-client-api/00-overview.md#transaction) that was used to retrieve the concept initially. As soon as the transaction in question is closed, methods can no longer be called on the retrieved Concepts. To continue using the Concept API on a concept, we must retrieve that concept again with a newly created transaction.
+In order to run Concept API methods, a live [transaction](../03-client-api/00-overview.md#transaction) is required (to execute the method RPCs). Concepts returned from a method will be bound to the same transaction as the concept that was used to call the method.
+
+Since Grakn 1.7.0, queries return some additional information about attribute values and types so that they do not need to be retreived using separate RPCs. Most clients will instead return a `Local` concept that is limited to retrieving this data. In order to make concept RPC calls, a `Remote` concept can be created from the `Local` one. See the client specific documentation for more details.
```

Comment:
- at the moment our documentation is based on the latest releases, so I think we can avoid phrases like since "since version x"

retreived > retrieved

> `so that they do not need to be retreived using separate RPCs`
and how does this benefit our users? I think we should mention that instead.

> Most clients will instead return a `Local` concept that is limited to retrieving this data
don't all clients?

> will instead return
use present tense when you can.